sorc/checkout.sh

```diff
@@ -158,7 +161,7 @@ checkout "ufs_utils.fd"    "https://github.com/ufs-community/UFS_UTILS.git"
 checkout "verif-global.fd" "https://github.com/NOAA-EMC/EMC_verif-global.git"   "c267780"                    ; errs=$((errs + $?))
 
 if [[ ${checkout_gsi} == "YES" ]]; then
-  checkout "gsi_enkf.fd"     "https://github.com/NOAA-EMC/GSI.git"         "48d8676"; errs=$((errs + $?))
+  checkout "gsi_enkf.fd" "https://github.com/NOAA-EMC/GSI.git" "48d8676" "NO"; errs=$((errs + $?))
```

Comment:
Two comments:
- NOAA-EMC/GSI develop is currently at [4a7c652](https://github.com/NOAA-EMC/GSI/commit/4a7c652d670f7aa5ad9890e88a89f2bf3011e618).   This hash brings in bug fixes and additional functionality.   While we may not want to update the GSI hash in this PR, it should be updated.
- The default behavior is to not clone GSI fix files.  This is fine as a default.  Bear in mind that DA developers may need to clone GSI fix files (e.g., testing new data or qc via changes to info file(s)).  They may not understand what `NO` means when they change the hash to clone.  An explanatory comment either here in the script would be useful.  Alternately the checkout script can print a message stating the source , staged or cloned, GSI fix files are being used.

Reply:
It is fair to add a blurb on the wiki.  The model fix files don't have a similar blurb either. We can add this to a "documentation" issue.

Reply:
@RussTreadon-NOAA 
Do the GSI code-managers intend to remove the git submodule to GSI-fix?

Reply:
Good point @aerorahul .  

The _YES/NO_ `recursive` option in `checkout.sh` will hopefully be short-lived.   

ASCII GSI fix files will likely move to a `parm` or `fix` directory in NOAA-EMC/GSI develop.   These are the files developers modify (add new obs, remove old obs, modify obs errors & qc, specify new met_guess, state/control vectors, etc.).  It's important to track changes in these files.   

Binary GSI fix files do not change very often.  These will live outside of the repo.  How best to store and version binary fix files applies to many EMC apps, not just the GSI.  We'll follow EIB's recommendation for a center-wide solution.

We do not have any GSI code-managers so I do not know when / who will remove the git submodule from GSI -fix.

Reply:
Do the GSI-admins intend to keep/remove the git submodule to GSI-fix?
If a decision to remove is made by the GSI-admins, I can offer to remove the git submodule pointer with a PR.

Reply:
Your question has been added to the agenda for the weekly GSI Repo checkup.